site/slices/contact-us-slice/form/index.js

```diff
@@ -15,6 +15,23 @@ class Form extends Component {
     };
   }
 
+  componentDidMount = () => {
+    const { stateNavigator } = this.context;
+    stateNavigator.onNavigate(this.focusText);
+  }
+
+  componentWillUnmount = () => {
+    const { stateNavigator } = this.context;
+    stateNavigator.offNavigate(this.focusText);
+  }
```

Comment:
We could possibly thread any query string params through to the component automatically here:
https://github.com/redbadger/website-honestly/blob/05f68187534b2ebbc9b3f106e148330d987c2349/site/client/index.js#L27

this would be easier to test and might be more robust?